src/main/java/frc/robot/commands/ArcadeDriveCommand.java

```diff
@@ -7,7 +7,6 @@
 import frc.robot.OI;
 import frc.robot.Constants.OperatorConstants;
 
-// import java.lang.Math.*;
```

Comment:
i would assume it does, because it was originally commented out before i edited anything, so it wasn't being used in the first place
but we might want to check with connie?
@ConnieJChi

Reply:
I'm pretty certain this code should compile without this line bc it's commented out. If it borks the robot I would be VERY surprised.